spec/index.html

```diff
@@ -392,7 +392,7 @@ <h3>Document Conventions</h3>
         <h3>Terminology</h3>
         <dl>
           <dt>SPARQL Protocol client</dt>
-          <dd>An HTTP client (as defined by <a data-cite="rfc9110#section-3.3">RFC 9110</a>) that sends HTTP requests for SPARQL
+          <dd>An HTTP client (as defined by <a data-cite="rfc9110#section-3.3">RFC 9110</a> [[RFC9110]]) that sends HTTP requests for SPARQL
           Protocol operations. (Also known as: <em>client</em>)</dd>
```

Comment:
Perhaps adopt this style? It will be clearer after processing by ReSpec, so there are two copies of the `<dd>` below -- one with my suggested change, followed by the version I'm suggesting be changed -- so they are more easily compared. This will require that this suggestion be merged, and then whichever style is chosen be applied throughout the document, and the other deleted here.
```suggestion
          <dd>An HTTP client (as defined by
            <a data-cite="rfc9110#section-3.3">Section 3.3. Connections, Clients, and
            Servers</a> of [[[RFC9110]]] [[RFC9110]]) that sends HTTP requests for
            SPARQL Protocol operations. (Also known as: <em>client</em>.)</dd>

          <dd>An HTTP client (as defined by <a data-cite="rfc9110#section-3.3">RFC 9110</a> [[RFC9110]]) that sends HTTP requests for SPARQL
          Protocol operations. (Also known as: <em>client</em>)</dd>
```